### ISSUE TITLE
Dark image when player HDR in Fullscreen after upgrade to Ventura

### ISSUE THREAD

**Issue by buntupana** [iina/iina#4014]
**System and IINA version:**

- macOS 13.0
- IINA 1.3.0
- Macbook pro 14 2021

**Expected behavior:**
Watch HDR content in Fullscreen

**Actual behavior:**
When I try to play a content in HDR in fullscreen the image will go dark when the controller display fade out. If I play no HDR content or I disable HDR in options this won't happen. I tried legacy and native fullscreen with the same result.

<br>

**Steps to reproduce:**

Play any HDR content

**How often does this happen?**

Always

**Comment by singh-jagjot** [comment]
The issue can be replicated on Macbook Pro 16 2021

**Comment by buntupana** [comment]
Just discoverd a workaround:

Keeping the pointer in the bottom right/left corner of the screen while in fullscreen will fix the problem

**Comment by low-batt** [comment]
Duplicate of #3844, #3880, #3906.

**Comment by Liferme** [comment]
> 

I don't think it is only a matter of HDR videos, I watched a normal 2160P Bluray content today, and the issue remains.

**Comment by singh-jagjot** [comment]
> > 
> 
> I don't think it is only a matter of HDR videos, I watched normal 2160P Bluray content today, and the issue remains.

For me, it's only limited to HDR videos, tested non-HDR(AV1, VP9) videos till 2160p and they are working as expected.

**Comment by royj** [comment]
I made a workaround here: https://github.com/iina/iina/pull/4017

I only tested it on my MBP16, so additional testing would be useful.

**Comment by Liferme** [comment]
@low-batt @singh-jagjot  **Sorry guys, I just updated to Ventura today, and I didn't do much testing when I commented.**

After a thorough test, it appears that we can confirm the issue is tied to HDR enabling in the quick settings panel. And it has nothing to do with hardware decoding.

**Behavior:**
The screen will become so dim that the video becomes unwatchable when the controller bar fade out in HDR fullscreen mode.

**My Device Specification:**
- macOS Ventura 13.0
- IINA 1.3.0
- Macbook pro 14 2021
